### DISCUSSION TITLE
FZF Shortcuts for Mac?

### DISCUSSION THREAD

**Discussion by ElijahLynn** [PatrickF1/fzf.fish#300] [Q&A] [answered]
Hi, Longtime user on Arch Linux here, love it!

I just had to start using a Mac for my dayjob, and am not able to use `ctrl + alt + f` for example to find files. This discussion may just be me rubber ducking in public. And if anyone else has suggestions on getting that working that would be great!

Cheers

**Comment by PatrickF1** [comment]
Hi Elijah, have you tried reading https://github.com/PatrickF1/fzf.fish/wiki/Troubleshooting#default-or-custom-key-bindings-do-not-work? From that list, a common one for Mac users is that iTerm, the most common terminal for Mac users, is intercepting the alt key and doing something else with it.

**Reply by ElijahLynn** [reply]
Oh nice Lead Patrick! I searched the readme for "mac" and was searching issue queue and discussions for "mac" too and didn't see any title hits so wasn't sure it was supported. I am using Iterm2, so definitely will give those issues a read through!

Thank you!!!

**Reply by ElijahLynn** [reply, marked as answer]
Sweet, found this from that article https://github.com/PatrickF1/fzf.fish/discussions/97 and then found https://stackoverflow.com/a/197092/292408 which fixed it and it works beautifully now!!!

Thanks

**Reply by ElijahLynn** [reply]
I do wonder about maybe adding a Mac or Iterm keyword to the README, or maybe saying that "Mac" should be supported. As a new Mac user (well, at least after 10 years of not using one) I kept seeing `ctrl` this and that and wasn't sure this extension supported Mac because I didn't see `cmd`. Not saying everyone is like that but that was the case for me at least. 

I can make a docs PR if you are interested in adding something.

**Reply by PatrickF1** [reply]
Great to hear! Would you mind starring the repo as a way of thanks?

**Reply by PatrickF1** [reply]
> I do wonder about maybe adding a Mac or Iterm keyword to the README, or maybe saying that "Mac" should be supported. As a new Mac user (well, at least after 10 years of not using one) I kept seeing ctrl this and that and wasn't sure this extension supported Mac because I didn't see cmd. Not saying everyone is like that but that was the case for me at least.

I don't have time to re-research the details but Command isn't supported by terminals/shells as a key modifier. Thanks for offering your perspective. I can make that edit myself

**Reply by ElijahLynn** [reply]
Starred ⭐ ✅ 

Cool, thanks for taking on the edit, I think you would be best suited to write it too!
